### PR TITLE
Milestone 2 user interface

### PR DESCRIPTION
**Pull request by brytebee**
This is a copy of the code below:

https://github.com/brytebee/tic_tac_toe/pull/2

It is done to comply with best practices on GITflow.

Please make an appropriate review, my game flow.

Thank you.

### CODE REVIEW

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @brytebee :wave: 

**It's obvious you've put a lot of effort into this project but there's just a little more to be done before this milestone can be approved.**

## REQUIRED
* [x] Your game is expected to ask players for their names. [Example screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone2/players_names.png)
* [x] Your game should inform players if the selected move is a winning move (without computing the winning move). [Example screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone2/game_win%2059.png)
* [x] Your game should inform players if the selected move is a draw move (without computing draw). [Example screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone2/game_tie.png)

#### See the gif below for more clarification on what is expected of you in this milestone
image

### Make the required changes and request another review.
## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Review by BrianSammit — APPROVED** [review]
STATUS: APPROVED :green_circle: :clap: 
-----------
Hello @brytebee  you did an amazing job on this milestone :muscle: I like how your code is clean, it's really simple to know what every line of code does. :clap: :smile: 
---------------
Now you can merge this branch and move to the next. 
CONGRATULATIONS!!! AND GOOD LUCK WITH THE NEXTS MILESTONES. :crossed_fingers: :four_leaf_clover:

**Pull request merged**
